config/initializers/_qiniu_sdk.rb

```diff
@@ -0,0 +1 @@
+Qiniu.establish_connection! Rails.application.secrets[:qiniu].symbolize_keys
```

Comment:
嗯,这个食失误,不用提前加载,应该是直接更新原来文件就好了                                                                                                                                                                        发送自我的 BlackBerry - 最安全的移动设备                                                                                                                                                                                                                                                          发件人：notifications@github.com发送时间：2016年5月23日 下午12:45收件人：jianggaowang@noreply.github.com回复：reply@reply.github.com抄送：hongzeqin@gmail.com; author@noreply.github.com主题：Re: [beansmile/jianggaowang] Optimize images (#240)  In config/initializers/_qiniu_sdk.rb:
> @@ -0,0 +1 @@
> +Qiniu.establish_connection! Rails.application.secrets[:qiniu].symbolize_keys

你是要让这个提前加载？那应该改名为001_qiniu_sdk.rb，并在顶部加入文件名这样命名的注释，不要用文件名_qiniu_sdk.rb

—You are receiving this because you authored the thread.Reply to this email directly or view it on GitHub